Cartfile

```diff
@@ -1,5 +1,5 @@
-github "pusher/pusher-platform-swift" ~> 0.7.1
-github "pusher/push-notifications-swift" ~> 2.0.2
+github "pusher/pusher-platform-swift" ~> 0.7.2
+github "pusher/push-notifications-swift" ~> 2.1.2
```

Comment:
It should be all fine but I would update `push-notifications-swift` dependency in the separate commit.
Because this PR is fixing race condition with `PPRepeater`, if something potentially goes wrong we don't know if it's a change in the platform SDK or Beams SDK.

Reply:
It is a separate commit 5147c155fafb62f1b158c12881e3d4f9005d5e19. Did you mean a separate pull request? I can submit it as a separate pull request. I just thought that because fixing `PPRepeater` is mostly a dependency update, I will also update other things.